teams/team_2/training/train.py

```diff
@@ -110,61 +135,77 @@ def __init__(self, features: torch.Tensor, state=None):
             self.state = torch.zeros(features.shape[0])
 
 
-population = [
-    (
-        TaskScorerNN(TASK_FEATURE_SIZE, PLAYER_STATE_SIZE, HIDDEN_SIZE),
-        # 1 is hardcoded
-        RestDecisionNN(PLAYER_STATE_SIZE + 1, HIDDEN_SIZE),
-    )
-    for _ in range(POP_SIZE)
-]
-
-avg_scores = []
-max_scores = []
+if __name__ == "__main__":
```

Comment:
these hunks below make it seem like I changed a whole lot but I really didn't. Don't look too much into it it's the same code.